src/client/providers/jediHelpers.ts

```diff
@@ -49,8 +49,8 @@ export function extractSignatureAndDocumentation(definition: proxy.IAutoComplete
 }
 
 export function highlightCode(docstring: string): string {
-    // Add blank line
-    docstring = EOL + EOL + docstring + EOL + EOL;
```

Comment:
Perhaps I didn't read through this closely enough before making changes to fix the tests up. My intention was to ensure the docstring was in a known state with exactly a certain amount of whitespace either end. I'll change it to have the two lines and do a trim at the end.

Also I'm curious - what's the circumstance that the docstrings would end with EOF?